packages/xo-web/src/common/xo/index.js

```diff
@@ -1879,6 +1880,20 @@ export const shareVm = async (vm, resourceSet) =>
     }),
   }).then(() => editVm(vm, { share: true }), noop)
 
+export const vmWarmMigration = async vm => {
+  const { sr, deleteSourceVm, startMigratedVm } = await confirm({
+    body: <WarmMigrationModal />,
+    title: _('vmWarmMigration'),
+    icon: 'vm-warm-migration',
+  })
+  return _call('vm.warmMigration', {
+    deleteSourceVm,
+    sr: resolveId(sr),
+    startDestinationVm: startMigratedVm,
```

Comment:
Not very important but since we're adding the feature: let's use `startDestinationVm` for both to simplify global searches, avoid typos, confusions between both etc.